### PR TITLE
Rewrite TestRunMutableNetworkFiles to avoid races

### PR DESCRIPTION
**Pull request by LK4D4**
ping @icecrime

### CODE REVIEW

File: integration-cli/docker_cli_run_test.go [hunk 1]
```diff
@@ -2016,16 +2016,16 @@ func TestRunMutableNetworkFiles(t *testing.T) {
 			t.Fatal("Content was not what was modified in the container", string(content))
 		}
 
-		out, _, err := runCommandWithOutput(exec.Command(dockerBinary, "run", "-d", "--name", "c2", "busybox", "sh", "-c", fmt.Sprintf("while true; do cat /etc/%s; sleep 1; done", fn)))
+		out, _, err := runCommandWithOutput(exec.Command(dockerBinary, "run", "-d", "--name", "c2", "busybox", "top"))
 		if err != nil {
 			t.Fatal(err)
 		}
 
 		contID := strings.TrimSpace(out)
```
Comment (icecrime) [inline integration-cli/docker_cli_run_test.go]: Do we need this `contID`? We already named it `"c2"` anyway.

File: integration-cli/docker_cli_run_test.go [hunk 2]
```diff
@@ -2007,7 +2007,7 @@ func TestRunMutableNetworkFiles(t *testing.T) {
 	for _, fn := range []string{"resolv.conf", "hosts"} {
 		deleteAllContainers()
 
-		content, err := runCommandAndReadContainerFile(fn, exec.Command(dockerBinary, "run", "-d", "--name", "c1", "busybox", "sh", "-c", fmt.Sprintf("echo success >/etc/%s; while true; do sleep 1; done", fn)))
+		content, err := runCommandAndReadContainerFile(fn, exec.Command(dockerBinary, "run", "-d", "--name", "c1", "busybox", "sh", "-c", fmt.Sprintf("echo success >/etc/%s && top", fn)))
```
Comment (icecrime) [inline integration-cli/docker_cli_run_test.go]: I don't think this container has to be long running? Maybe `"echo [...] && cat /etc/%s"` is enough?

File: integration-cli/docker_cli_run_test.go (hunk 1 above)
Comment (LK4D4) [inline integration-cli/docker_cli_run_test.go]: Yeah, we need :/ `containerStorageFile` requires id.

Comment (icecrime) [inline integration-cli/docker_cli_run_test.go]: Oh right, sorry.

File: integration-cli/docker_cli_run_test.go (hunk 2 above)
Comment (LK4D4) [inline integration-cli/docker_cli_run_test.go]: I really think that it's better to check this on running container.

**Comment by LK4D4** [comment]
@icecrime @jfrazelle WDYT?

**Comment by icecrime** [comment]
LGTM

**Comment by jfrazelle** [comment]
LGTM

**Pull request merged**